source/_integrations/gtfs.markdown

```diff
@@ -62,7 +62,7 @@ name:
   default: GTFS Sensor
   type: string
 offset:
-  description: A minimum delay to look for. If a departure is in less time than `offset`, it will be ignored.
+  description: A minimum delay to look for. If a departure is in less time than `offset`, it will be ignored. Set the value either in seconds (`integer`), or using one of these `time` formats: `offset: 'HH:MM:SS'`, `offset: 'HH:MM'`. 
```

Comment:
```suggestion
  description: "A minimum delay to look for. If a departure is in less time than `offset`, it will be ignored. Set the value either in seconds (`integer`), or using one of these `time` formats: `offset: 'HH:MM:SS'`, `offset: 'HH:MM'`. "
```